CHANGELOG.md

```diff
@@ -14,6 +14,10 @@ Released 2025-02-10
  - Added support for the Arm Debug Interface v6 (ADIv6). Support added for flashing RP235x targets.
    Development sponsored by osdyne. (osdyne.com) (#3005) by @ryan-summers
  - Add a function to reinitialize the connection to the Black Magic Probe. This goes through the entire initialization process with the same sequence as it was originally started with. (#3032) by @xobs
+ - Added support for the BCM2712 target (Raspberry Pi 5) (#2209) by @tnishinaga
+ - Added support for the CH32V305 variant (#2857) by @Dummyc0m
+ - Added support for MSPM0Gx targets (#2868) by @optlink
+ - Added support for the STM32WBA (#1761) by @Dirbaio
```

Comment:
Aren't these ancient?

Reply:
Yeah I was following the release MD file and these were all the PRs listed. A quick skim of the changelog didn't show them elsewhere, so I'm not sure how to order them

Reply:
I reverted this commit and removed these entries

Reply:
I think that's fair. We can maybe find their place but for year old things it just might not matter.